doc/source/development/contributing_gitpod.rst

```diff
@@ -0,0 +1,282 @@
+.. contributing-gitpod:
+
+Using Gitpod for pandas development
+==================================
+
+This section of the documentation will guide you through:
+
+*  using GitPod for your pandas development environment
+*  creating a personal fork of the pandas repository on GitHub
+*  a quick tour of pandas and VSCode
+*  working on the pandas documentation in Gitpod
+
+Gitpod
+------
+
+`Gitpod`_  is an open-source platform for automated and ready-to-code
+development environments. It enables developers to describe their dev
+environment as code and start instant and fresh development environments for
+each new task directly from your browser. This reduces the need to install local
+development environments and deal with incompatible dependencies.
+
+
+Gitpod GitHub integration
+-------------------------
+
+To be able to use Gitpod, you will need to have the Gitpod app installed on your
+GitHub account, so if
+you do not have an account yet, you will need to create one first.
+
+Head over to the `Gitpod`_ website and click on the **Continue with GitHub**
+button. You will be redirected to the GitHub authentication page.
+You will then be asked to install the `Gitpod GitHub app <https://github.com/marketplace/gitpod-io>`_.
+
+Make sure to select **All repositories** access option to avoid issues with
+permissions later on. Click on the green **Install** button
+
+.. image:: ./gitpod-imgs/installing-gitpod-io.png
```

Comment:
This image seems to be missing in the branch? (and some of the others as well, there are only 2 images committed)